lefthk-core/src/config/command/utils/error.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(thiserror::Error, Debug, Clone, PartialEq, Eq, Hash)]
+pub enum CommandError {
+    #[error("Given String doesn't match.")]
+    UnmatchingCommand,
+}
```

Comment:
Is there any point to having this here, especially when there is only one error

Reply:
This is just a provision if we need more errors in the context of a command. But we can remove this if you want, since there's only one error (yet?).

Reply:
Yeah, I would say keep it in one place for now. This would be more helpful on bigger projects, where this sub-error file would have like 6 plus error in it.